guides/introduction.mdx

```diff
@@ -7,6 +7,42 @@ These guides cover key aspects of building and deploying Pipecat applications. C
 
 ## Features
 
+<CardGroup cols={2}>
```

Comment:
This is duplicating the Features section

Reply:
Duh, sorry, done. I was going to add the AI models section.